release_images.yml

```diff
@@ -3,50 +3,66 @@ release_images:
 # Check .release_images_template.yml for reference
   1:
     framework: "pytorch"
-    version: "1.10.2"
+    version: "1.12.1"
     arch_type: "x86"
+    customer_type: "sagemaker"
+    training:
+      device_types: [ "cpu", "gpu" ]
+      python_versions: [ "py38" ]
+      os_version: "ubuntu20.04"
+      cuda_version: "cu116"
```

Comment:
cuda version should be `cu113` for SM and `cu116` for E3

Reply:
ref uri in https://github.com/aws/deep-learning-containers/blob/master/available_images.md